### ISSUE TITLE
Unknown type name `BulletinManager` in Objective-C

### ISSUE THREAD

**Issue by Sjoerdjanssenen** [alexaubry/BulletinBoard#48]
I installed the library through CocoaPods and then imported it in the viewcontroller as shown in the example:

    @import BulletinBoard;

Then when I create the property for `BulletinManager`, Xcode will throw an error `Unknown type name BulletinManager`.

### New Issue Checklist
- [x] I updated BulletinBoard to the latest version.
- [x] I read the [Contribution Guidelines](https://github.com/alexaubry/BulletinBoard/blob/master/CONTRIBUTING.md).
- [x] I read the [documentation](https://alexaubry.github.io/BulletinBoard).
- [x] I searched for [existing GitHub issues](https://github.com/alexaubry/BulletinBoard/issues).

### Issue Description
I installed the library through CocoaPods and then imported it in the viewcontroller as shown in the example:

    @import BulletinBoard;

Then when I create the property for `BulletinManager`, Xcode will throw an error `Unknown type name BulletinManager`.
<img width="1075" alt="screen shot 2017-12-29 at 00 41 46" src="https://user-images.githubusercontent.com/8407498/34425892-1ea78152-ec31-11e7-9759-53eb962953b8.png">

### Environment

- **iOS Version**: 11.2
- **Device(s)**: iPhone X

**Comment by alexaubry** [comment]
The version that supports Objective-C has not been released yet (there are still some bugs to fix). If you need to use BulletinBoard before the new version is released, you can set your Podfile to use the master branch:

~~~ruby
pod 'BulletinBoard', :git => 'https://github.com/alexaubry/BulletinBoard.git', :branch => 'master'
~~~

**Comment by rizwan95** [comment]
+1 ! I also need an objective C version.

**Comment by alexaubry** [comment]
@rizwan95 You can try the beta version of the framework to get the Objective-C API.

**Comment by alexaubry** [comment]
The Objective-C version is available in the version 2.0.0 of the library. You can update using the instructions in the release notes.
